segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -240,10 +248,21 @@ SnapshotInfoStore getStorageSnapshotInfoStore() {
                 () -> readStorageSnapshot(config.getStorageSnapshotTimeout()));
     }
 
+    private Function<String, CompletableFuture<DirectSegmentAccess>> getOrCreateContainerProcessorSegment(int containerId,
+                                                                                                          Duration duration) {
+        final SegmentType systemCritical = SegmentType.builder().system().critical().build();
```

Comment:
add `internal()` as well. In the future, we will be enforcing that we won't allow external access to internal segments (unless via the admin gateway)

Reply:
Added.